game_functions.cpp

```diff
@@ -42,6 +42,12 @@ double get_nth_player_size(const game& g, const int i)
   return g.get_player(i).get_diameter();
 }
 
+
+color get_nth_player_color(const game& g, const int i)
```

Comment:
I think it is done perfectly

Reply:
Agreed, except on one point: the always dreaded const correctness ;)
Since this function doesn't require to change anything in either the input or output, [it should be therefore labelled](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#con1-by-default-make-objects-immutable) `const` ;)

```suggestion
const color get_nth_player_color(const game& g, const int i)
```

Reply:
Problem is that:
 `const color` in  .cpp, but `color` in the header file.